brim/zeek

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+file=$(realpath "${BASH_SOURCE[0]}")
```

Comment:
I don't have `realpath` in my path on my mac; I think that's from gnu coreutils? I'm used to seeing it on linux.